x-pack/plugins/ingest_manager/public/applications/ingest_manager/components/enrollment_instructions/manual/index.tsx

```diff
@@ -11,7 +11,7 @@ import { FormattedMessage } from '@kbn/i18n/react';
 import { EnrollmentAPIKey } from '../../../types';
 
 interface Props {
-  kibanaUrl: string;
+  kibanaUrls: string;
```

Comment:
it seems that the expected prop still comes in as a single string, so do we need to pluralize the prop name here?

Reply:
Oops, good catch.  No we don't because we only provide one url at the moment until the cli supports a comma separated list.